internal/installer/elasticagent_tar_macos.go

```diff
@@ -79,8 +86,13 @@ func (i *elasticAgentTARDarwinPackage) Exec(ctx context.Context, args []string)
 }
 
 // Enroll will enroll the agent into fleet
+<<<<<<< HEAD
 func (i *elasticAgentTARDarwinPackage) Enroll(ctx context.Context, token string) error {
 	cmds := []string{common.GetElasticAgentWorkingPath("elastic-agent"), "install"}
+=======
+func (i *elasticAgentTARDarwinPackage) Enroll(ctx context.Context, token string, extraFlags string) error {
+	cmds := []string{"sudo", common.GetElasticAgentWorkingPath("elastic-agent", "elastic-agent"), "install"}
+>>>>>>> 892e951c (Use Orka ephemeral workers to run the fleet testing (#2626))
```

Comment:
Extraflags appeared in 8.4
```suggestion
func (i *elasticAgentTARDarwinPackage) Enroll(ctx context.Context, token string) error {
	cmds := []string{"sudo", common.GetElasticAgentWorkingPath("elastic-agent"), "install"}
```